tox.ini

```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-complexity = 10
+max-line-length = 100
```

Comment:
I would vote for this, and make build fail if these are not met to keep things tidy. But I am curious what your opinion is?

Reply:
Complexity I'm happy with, but ```max-line-length``` can be frustrating for very little gain in readability. Then again, 100 is probably fine (80 is absolutely awful). I think there are cases where a line could be a bit longer and be _clearer_ than when it gets split over several lines, and that's fine as long as it is only occasional. I wish you could have it fail if too many lines are really long...

Reply:
OK, thinking about it, let's keep your change as is, and change back if line length gets too bad.

Reply:
Yeah can always add an `ignore` comment if you have complexity complaints that you don't want to fix.

I am raised with line-length of 80, so 100 already hurts my eyes 😛 🙈 I believe it keeps everything tidy, and if you're used to it it is no effort at all. But indeed let's change it back if it bothers further development!